src/setupTests.js

```diff
@@ -0,0 +1,4 @@
+import { configure } from "enzyme"
+import Adapter from "enzyme-adapter-react-16"
+
+configure ({ adapter: new Adapter() })
```

Comment:
not sure if important but I would skip the space between `configure` and the parenthesis

Reply:
Always good to follow to follow best practices with formatting, thanks for pointing it out!  I've removed the extra space now